ROADMAP.md

```diff
@@ -2,60 +2,49 @@
 
 ## About this Document
 
-This document provides an overview of the major themes driving Tanzu Community Edition development, as well as some of the features and
-capabilities that will manifest those priorities, grouped into 0-3 month, 3-6 month, and future buckets. Most items are gathered from
-the community or include a feedback loop with the community. This should serve as a reference point for Tanzu Community Edition users
-to help us prioritize and to supply input on unmet needs, and contributors to understand where the project is heading, and help
-contributors to determine if a contribution could be conflicting with a longer term plan.
+This document provides an overview of the major themes driving Tanzu Community Edition program development, as well as constituent features and capabilities. Roadmap items are tracked by their implementation stage within the enhancements proposal process: Awaiting Proposal, Proposal in Review, and Implementation in Progress. The stages are documented below. Most items are gathered from the community or include a feedback loop with the community.
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+The scope of this roadmap is the entire Tanzu Community Edition program, which includes this GitHub project repository, the [Tanzu Framework](https://github.com/vmware-tanzu/tanzu-framework) project frepository, the project repositories for packages delivered as part of Tanzu Comm unity Edition, and non-code program aspects such as videos, training materials, and community activities.
```

Comment:
> The scope of this roadmap is the entire Tanzu Community Edition program, which includes this GitHub project repository, the **Tanzu Framework project repository**

Emphasis mine. Does this mean we are claiming and owning the roadmap for Framework?